catalyst-gateway/bin/src/event_db/voter_registration.rs

```diff
@@ -1,32 +1,36 @@
 //! Voter registration queries
 
-use super::{Error, EventDB};
+use cardano_chain_follower::Network;
+use pallas::ledger::traverse::MultiEraTx;
+use serde_json::{json, Value};
+
+use super::{follower::SlotNumber, Error, EventDB};
+use crate::registration::{
+    parse_registrations_from_metadata, validate_reg_cddl, CddlConfig, Nonce as NonceReg,
+};
 
 /// Transaction id
 pub(crate) type TxId = String;
 /// Stake credential
 pub(crate) type StakeCredential<'a> = &'a [u8];
 /// Public voting key
-pub(crate) type PublicVotingKey<'a> = &'a [u8];
+pub(crate) type PublicVotingKey = String;
```

Comment:
As inside the query itself you are taking just bytes `public_voting_key.as_bytes()` seems it is make sense leave this type as `&'a [u8]`. WDYT ?

Reply:
good point, changed.